### PR TITLE
DOC-4382: Update screebshots for Allow to add spaces with dynamic layout

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Administration.rst [hunk 1]
```diff
@@ -3069,9 +3069,13 @@ for more details.
 | Show Max Window        | The option enables the page to be shown at the    |
 |                        | maximum size or not.                              |
 +------------------------+---------------------------------------------------+
+| Use dynamic layout     | The option enables the use of dynamic layout      |
+|                        | or not.                                           |
```
Comment (boubaker) [inline docs/Administration.rst]:
```suggestion
| Use dynamic layout     | When enabled, the group (can be a space) site layout |
|                        | will be retrieved from current PORTAL site layout. |
|                        | This options will be turned off automatically |
|                        | if the user edits the group site |
|                        | layout and saves changes. |
|                        | To turn it on again, you should edit the group site |
|                        | properties then enable the checkbox and save changes. |
```

File: docs/Administration.rst [hunk 2]
```diff
@@ -3069,9 +3069,18 @@ for more details.
...
+|                        | if the user edits the group site |
+|                        | layout and saves changes. |
+|                        | To turn it on again, you should edit the group site |
+|                        | properties then enable the checkbox and save changes. |
++------------------------+---------------------------------------------------+
 
-3. Define the page layout in the Page Layout tab.
 
+3. Define the page layout in the Page Layout tab.
+s
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion

```

**Review by plamarque — CHANGES_REQUESTED** [review]
just a small typo. Otherwise OK.

**Comment by hghorbel** [comment]
Typo problem resolved.

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**